### PR TITLE
Refine environment variables name

### PR DESCRIPTION
**Pull request by abuccts**
Refine enironment variables name. Refer to #322 and #325.

### CODE REVIEW

File: job-tutorial/README.md [hunk 1]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
 | PAI_OUTPUT_DIR                     | `outputDir`in config file or the generated path if `outputDir` is not specified |
 | PAI_CODE_DIR                       | `codeDir` in config file                 |
-| PAI_TASK_ROLE_LIST                 | Comma separated all task role names in config file |
 | PAI_TASK_ROLE_NAME                 | `taskRole.name` of current task role     |
-| PAI_TASK_ROLE_NUM                  | `taskRole.taskNumber` of current task role |
-| PAI_TASK_CPU_NUM                   | `taskRole.cpuNumber` of current task     |
-| PAI_TASK_MEM_MB                    | `taskRole.memoryMB` of current task      |
-| PAI_TASK_GPU_NUM                   | `taskRole.gpuNumber` of current task     |
-| PAI_TASK_ROLE_INDEX                | Index of current task in the task role, starting from 0 |
+| PAI_TASK_ROLE_TASK_NUM             | `taskRole.taskNumber` of current task role |
```
Comment (fanyangCS) [inline job-tutorial/README.md]: >PAI_TASK_ROLE_TASK_NUM [](start = 2, length = 22)

I prefer NUM_OF_TASK as it is more clear.

File: job-tutorial/README.md [hunk 2]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
+| PAI_TASK_ROLE_CPU_NUM              | `taskRole.cpuNumber` of current task role  |
+| PAI_TASK_ROLE_MEM_MB               | `taskRole.memoryMB` of current task role   |
+| PAI_TASK_ROLE_GPU_NUM              | `taskRole.gpuNumber` of current task role  |
+| PAI_TASK_ROLE_TASK_INDEX           | Index of current task in the task role, starting from 0 |
 | PAI_TASKS_NUM                      | Total tasks' number in config file       |
```
Comment (sterowang) [inline job-tutorial/README.md]: PAI_JOB_TASK_COUNT

File: job-tutorial/README.md (hunk 1 above)
Comment (sterowang) [inline job-tutorial/README.md]: PAI_CURRENT_TASK_ROLE_TASK_COUNT

File: job-tutorial/README.md [hunk 3]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
-| PAI_TASK_ROLE_NUM                  | `taskRole.taskNumber` of current task role |
-| PAI_TASK_CPU_NUM                   | `taskRole.cpuNumber` of current task     |
-| PAI_TASK_MEM_MB                    | `taskRole.memoryMB` of current task      |
-| PAI_TASK_GPU_NUM                   | `taskRole.gpuNumber` of current task     |
-| PAI_TASK_ROLE_INDEX                | Index of current task in the task role, starting from 0 |
+| PAI_TASK_ROLE_TASK_NUM             | `taskRole.taskNumber` of current task role |
+| PAI_TASK_ROLE_CPU_NUM              | `taskRole.cpuNumber` of current task role  |
+| PAI_TASK_ROLE_MEM_MB               | `taskRole.memoryMB` of current task role   |
+| PAI_TASK_ROLE_GPU_NUM              | `taskRole.gpuNumber` of current task role  |
+| PAI_TASK_ROLE_TASK_INDEX           | Index of current task in the task role, starting from 0 |
```
Comment (sterowang) [inline job-tutorial/README.md]: PAI_CURRENT_TASK_ROLE__CURRENT_TASK_INDEX

File: job-tutorial/README.md [hunk 4]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
 | PAI_TASKS_NUM                      | Total tasks' number in config file       |
 | PAI_TASK_ROLES_NUM                 | Total task roles' number in config file  |
+| PAI_TASK_ROLES_LIST                | Comma separated all task role names in config file |
 | PAI_KILL_ALL_ON_COMPLETED_TASK_NUM | `killAllOnCompletedTaskNumber` in config file |
-| PAI_CURRENT_CONTAINER_IP           | Allocated ip for current docker container |
-| PAI_CURRENT_CONTAINER_PORT         | Allocated port for current docker container |
+| PAI_CONTAINER_IP                   | Allocated ip for current docker container |
```
Comment (sterowang) [inline job-tutorial/README.md]: PAI_CONTAINER_HOST_IP

File: job-tutorial/README.md [hunk 5]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
+| PAI_TASK_ROLE_CPU_NUM              | `taskRole.cpuNumber` of current task role  |
+| PAI_TASK_ROLE_MEM_MB               | `taskRole.memoryMB` of current task role   |
+| PAI_TASK_ROLE_GPU_NUM              | `taskRole.gpuNumber` of current task role  |
+| PAI_TASK_ROLE_TASK_INDEX           | Index of current task in the task role, starting from 0 |
 | PAI_TASKS_NUM                      | Total tasks' number in config file       |
 | PAI_TASK_ROLES_NUM                 | Total task roles' number in config file  |
+| PAI_TASK_ROLES_LIST                | Comma separated all task role names in config file |
 | PAI_KILL_ALL_ON_COMPLETED_TASK_NUM | `killAllOnCompletedTaskNumber` in config file |
-| PAI_CURRENT_CONTAINER_IP           | Allocated ip for current docker container |
-| PAI_CURRENT_CONTAINER_PORT         | Allocated port for current docker container |
```
Comment (sterowang) [inline job-tutorial/README.md]: PAI_CONTAINER_HOST_PORT

File: job-tutorial/README.md [hunk 6]
```diff
@@ -134,22 +134,22 @@ Below we show a complete list of environment variables accessible in a Docker co
...
+| PAI_CURRENT_TASK_ROLE_CURRENT_TASK_INDEX | Index of current task in current task role, starting from 0 |
+| PAI_JOB_TASK_COUNT                 | Total tasks' number in config file       |
+| PAI_JOB_TASK_ROLE_COUNT            | Total task roles' number in config file  |
+| PAI_JOB_TASK_ROLE_LIST             | Comma separated all task role names in config file |
 | PAI_KILL_ALL_ON_COMPLETED_TASK_NUM | `killAllOnCompletedTaskNumber` in config file |
-| PAI_CURRENT_CONTAINER_IP           | Allocated ip for current docker container |
-| PAI_CURRENT_CONTAINER_PORT         | Allocated port for current docker container |
+| PAI_CONTAINER_HOST_IP              | Allocated ip for current docker container |
+| PAI_CONTAINER_HOST_PORT            | Allocated port for current docker container |
 | PAI_TASK_ROLE\_`$name`\_HOST_LIST  | Host list for `PAI_TASK_ROLE_NAME == $name`, comma separated `ip:port` string |
```
Comment (fanyangCS) [inline job-tutorial/README.md]: >| [](start = 117, length = 1)

please add "sorted by task role name and then task role index"

**Pull request merged**